packages/cli/src/lib/decorators/preconditions/ng.ts

```diff
@@ -10,6 +10,7 @@ export const IsNgVersionInRange = getBinVersionPrecondition(
     howToInstallBinText:
       'You can install the Angular-CLI by running npm i -g @angular/cli',
     installLink: 'https://angular.io/guide/setup-local#install-the-angular-cli',
+    params: ['version'],
```

Comment:
No more --version flag in 13. 13 and 14 support `ng version`, so let's use that. (no idea for before, but well, gee)